js/index.js

```diff
@@ -53,8 +53,17 @@ let mySelector4 = document.querySelector('#cta-img');
 
 const cta = document.querySelector('#cta-img');
 cta.setAttribute('src', siteContent['cta']['img-src']);
-console.log(cta);
+//console.log(cta);
 
 const middleImg = document.querySelector('#middle-img');
 middleImg.setAttribute('src', siteContent['main-content']['middle-img-src']);
-console.log(middleImg);
\ No newline at end of file
+//console.log(middleImg);
+
+// Updating the nav section
+
+let nav = document.querySelectorAll('nav a');
+//nav.forEach(e => {e.textContent = siteContent['nav'][nav-item-1]});
+nav.forEach(function(element, index){
+  element.textContent = siteContent['nav']['nav-item-'+ index];
+});
```

Comment:
nice way of looping and  adding content in a dynamic way, it make your code more dry